modules/ROOT/pages/index.adoc

```diff
@@ -1,88 +1,132 @@
-= MuleSoft Documentation Resources
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
-:page-aliases: general:getting-started:index.adoc
+= [.brand]#MuleSoft# Documentation
+:page-layout: home
+:page-fragmentize:
+:!sectids:
+ifndef::env-site[:imagesdir: ../images]
 
-Your customers and employees need data-rich, delightful digital experiences on a variety of devices from smart watches to desktop computers. To deliver these experiences, your systems must be connected to each other, and the data must flow among those systems (integration).
+[#cta]
+xref:api-led-overview.adoc[New to MuleSoft? Start here]
 
-The MuleSoft approach to integration, https://blogs.mulesoft.com/dev/api-dev/what-is-api-led-connectivity/[integration of data from different systems using a layer of APIs], allows you to spend less time on IT projects and more time on your core business. Whenever you turn a business process into an API, you make it easier to implement that process in the next project.
+[#the-road]
+== The Road to Digital Transformation
 
-image::api-led-architecture.png[Diagram of structured API layers for experience, process, and system APIs that can be easily added or removed]
+image::getting-started.png[]
 
-Anypoint Platform enables you to build a structured application network that uses reusable APIs to connect applications, data, and devices.
-Anypoint Platform makes it easy to discover, create, and manage APIs in a modular, organized layer. Instead of retrieving random (and possibly unstable) code snippets, you can discover and use APIs that are created using the industry’s best practices.
+--
+[discrete]
+=== Getting Started
 
-== Exploring Anypoint Exchange
+Build the digital transformation your business needs.
 
-The APIs you build in Anypoint Platform to integrate applications and services are reusable and include enterprise-level security.
-You can discover these APIs, as well as connectors, samples, and templates, in
-https://anypoint.mulesoft.com/exchange/[Exchange^].
+* xref:api-led-overview.adoc[Tutorial: Build an API from Start to Finish]
+--
 
-Exchange also provides RAML fragments, custom packages, videos, links to documentation, and other assets.
+image::discover.png[]
```

Comment:
Do we need alt text for these images? A11Y?

Reply:
added alt text